Persistence/Database/SeedData.cs

```diff
@@ -0,0 +1,175 @@
+﻿using Microsoft.EntityFrameworkCore;
+using OfficeOpenXml;
+
+using shipping_service.Extensions;
+using shipping_service.Persistence.Entities;
+
+namespace shipping_service.Persistence.Database
+{
+    public class SeedData
+    {
+
+        private const bool seed = true;
+        private const bool migrate = true;
+        private const string basePath = $"{nameof(Persistence)}/{nameof(Database)}/test-data";
+
+        public static void PopulateIfEmpty(IApplicationBuilder app)
+        {
+            ExcelPackage.LicenseContext = LicenseContext.NonCommercial;
+            DatabaseContext context = app.ApplicationServices.CreateScope().ServiceProvider
+                .GetRequiredService<DatabaseContext>();
+            if (migrate && context.Database.GetPendingMigrations().Any())
+            {
+                context.Database.Migrate();
+            }
```

Comment:
Migrations are already done in Program.cs. Currently migrations are always done, but control can be added. Also, this is not a good place to do migrations.

Reply:
Haven't seen it. Approved.